runtime/vm/ClassInitialization.cpp

```diff
@@ -796,6 +796,14 @@ classInitStateMachine(J9VMThread *currentThread, J9Class *clazz, J9ClassInitStat
 					clazz = VM_VMHelpers::currentClass(clazz);
 					goto done;
 				}
+#if defined(J9VM_OPT_VALHALLA_STRICT_FIELDS)
+				/* All strict static fields must be set by the end of <clinit> */
```

Comment:
Please consistently mention modifiers in canonical order and finish the sentence with a period.
```c
				/* All static strict fields must be set by the end of <clinit>. */
```